com.ibm.wala.scandroid/source/org/scandroid/flow/InflowAnalysis.java

```diff
@@ -54,13 +54,7 @@
 
 //import static util.MyLogger.LogLevel.DEBUG;
 
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Iterator;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
Star import

Reply:
I agree. My IDE was trying to be helpful behind my back. I have reconfigured my IDE and will correct this in a forthcoming group of updates for this pull request.